src/vs/workbench/services/search/node/ripgrepTextSearchEngine.ts

```diff
@@ -42,6 +42,9 @@ export class RipgrepTextSearchEngine {
 
 			const cwd = options.folder.fsPath;
 
+			console.log({ query, options, rgArgs, cwd });
```

Comment:
👉

Reply:
oops

Reply:
Wait I removed that already, I think you're looking at an old version

Reply:
Ah I think the gh notification will just take to some random new commit which is not necessarily the latest one